test/integration/utils.go

```diff
@@ -96,3 +100,57 @@ func GetEtcdClients(config storagebackend.TransportConfig) (*clientv3.Client, cl
 
 	return c, clientv3.NewKV(c), nil
 }
+
+func StartTestAPIServer(t *testing.T) (*kubeapiservertesting.TestServer, *os.File, string, error) {
```

Comment:
`StartTestServerOrDie` can mock a test server, so I think maybe we don't need a new function here.

Reply:
ditto https://github.com/kubernetes/kubernetes/pull/118148#discussion_r1212254598